### ISSUE TITLE
Unable to share attachments or large pastes

### ISSUE THREAD

**Issue by deeso** [jgeusebroek/docker-privatebin#12]
When attempting to share large pastes or files, an error occurs in nginx.  Its unable to create a file in the `/var/lib/nginx/tmp/client_body/`  The error might be related to permissions: 
1. https://wincent.com/wiki/Fixing_nginx_client_body_temp_permission_denied_errors

Here is a sample of the log output using `docker logs`:
```
2020/10/07 17:18:33 [crit] 14#14: *1 open() "/var/lib/nginx/tmp/client_body/0000000001" failed (13: Permission denied), client: 1.1.1.5, server: , request: "POST / HTTP/1.1", host: "privatebin.XXX.org"

```

I have spent a few hours trying to get it to work, but not been successful.

**Comment by jgeusebroek** [comment]
Thanks for the report. How large does the paste have to be to replicate this? What have you already tried to fix it?

**Comment by deeso** [comment]
The paste I was dealing with was ~1500 line shell script.  

In a very hodge-podge way, I attempted to modify the permissions of the directory so that they would correlate with Docker.  I also attempted to mount a folder volume in place of the directory with world read/write access.  I also tried modifying the container's `nginx` configuration with a bigger size.  I was in a rush to try and make it work, so I wasn't very methodical.  Given my lack of methodology, I would say that it might be best to ignore my work.
